optax/_src/transform_test.py

```diff
@@ -202,6 +202,35 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  @parameterized.parameters(
+    (0.01, 0.55, 123),
+    (0.3, 0.54, 234),
+    (1.0, 0.56, 314),
+  )
+  def test_add_noise(self, eta, gamma, seed):
```

Comment:
Could you provide more information on the behaviour being tested? Either by changing the name to e.g. `test_add_noise_has_correct_standard_deviation` or by adding a docstring with something similar (just so that there is more information if the test fails). Thanks a lot!

Reply:
I've gone with `test_add_noise_has_correct_variance_scaling` since the test doesn't actually check the standard deviation, just the scale relative to eta=1, gamma=0.
Thanks for all the nice comments.